scripts/funcotator/data_sources/gnomAD/liftoverVcf_Gnomad2.1.json

```diff
@@ -0,0 +1,19 @@
+{
+  "LiftoverVcf.gatk_docker": "broadinstitute/gatk:4.0.11.0",
+
+  "LiftoverVcf.variant_vcfs": [ "gs://gnomad-public/release/2.1/vcf/exomes/gnomad.exomes.r2.1.sites.chrX.vcf.bgz" ],
+  "LiftoverVcf.chain_file": "gs://broad-dsde-methods-jonn/b37ToHg38.over.chain",
```

Comment:
You should move this file somewhere accessible to the public, right?  This bucket is private.

Reply:
This bucket is actually a public bucket from which all the gnomAD data can be downloaded.

Reply:
The liftover file is now checked into the codebase.  I don't know which bucket is appropriate to house it.  I will reach out to comms to let them know this file exists and they can put it somewhere appropriate.

In general, these json files are preserved here to show how the liftover was run.  In the past I have had trouble verifying the provenance of data sets.  Preserving these files is one way to keep it.